apps/api/v2/src/modules/teams/memberships/inputs/get-team-memberships.input.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiPropertyOptional } from "@nestjs/swagger";
+import { Transform } from "class-transformer";
+import { IsOptional, Validate, ArrayMaxSize } from "class-validator";
+
+import { GetUsersInput } from "../../../users/inputs/get-users.input";
```

Comment:
It's better to use absolute path `"@/modules/users/inputs/get-users.input";` and same for `IsEmailStringOrArray` unless the file is in the same folder.